ansible/roles/common/tasks/debian.yml

```diff
@@ -25,10 +25,20 @@
   retries: 5
   until: apt_install_common is success
 
-- name: install common Kubernetes ansible module
-  pip:
-    name:
-      - openshift
+- name: 'add apt key for openshift kubernetes module (1/3)'
+  ansible.builtin.apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: ADF016E51783213C8A8C831A1348C52605DB2FE6
+
+- name: 'add OBS repo for openshift kubernetes module (2/3)'
+  ansible.builtin.apt_repository:
+    repo: deb http://download.opensuse.org/repositories/home:/anthr76:/kubernetes/Ubuntu_20.04/ /
+    state: present
+
+- name: 'install openshift kubernetes module (3/3)'
+  apt:
+    name: python-openshift
```

Comment:
Do we use this anywhere?

Reply:
We do in Cilium. https://github.com/raspbernetes/k8s-cluster-installation/blob/0586a6f19a809c0669e125303eaae0bfccf1b80f/ansible/roles/cni/tasks/cilium.yml#L24-L54

Likely elsewhere if the community is in agreeance, and sees fit for other CNI deployments etc

Reply:
commit: https://github.com/raspbernetes/k8s-cluster-installation/pull/72/commits/0c527dca735951b135de0e737b466bc61cbc17cd

Reply:
I wonder if this is purely in usage for the CNI should we keep it here or move it to the respective role?

Reply:
Good question. If we slot in CNI role we'll have to do distro based checks. The equivalent to this for Arch is also in common. In my opinion it would make sense here.

Reply:
Just to add on to this tidbit. If we were to move forward with issues like [this](https://github.com/raspbernetes/k8s-cluster-installation/issues/47) it would make more sense to keep this in common.